datalad/interface/tests/test_save.py

```diff
@@ -291,7 +291,7 @@ def check_renamed_file(recursive, no_annex, path):
         ds = Dataset(path).create(no_annex=no_annex)
         create_tree(path, {'old': ''})
         ds.add('old')
-        ds.repo._git_custom_command(['old', 'new'], ['git', 'mv'])
+        ds.repo.call_git(['git', 'mv'], files=['old', 'new'])
```

Comment:
Guess you figured it, but here you cause the test failures by passing a superfluous `'git'`.